addons/mail/static/src/models/clock/clock.js

```diff
@@ -0,0 +1,64 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, many } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'Clock',
+    identifyingFields: ['messaging', 'frequency'],
+    lifecycleHooks: {
+        _willDelete() {
+            this.messaging.browser.clearInterval(this.intervalId);
+        },
+    },
+    recordMethods: {
+        /**
+         * @private
+         * @returns {number}
+         */
+        _computeIntervalId() {
+            if (this.watchers.length > 0) {
+                return this.messaging.browser.setInterval(this._onTimeout, this.frequency);
+            } else {
+                this.messaging.browser.clearInterval(this.intervalId);
+                return clear();
+            }
+        },
+        /**
+         * @private
+         */
+        _onTimeout() {
```

Comment:
`_onInterval`

Reply:
Also this should `{ interval: clear() }`, just to make sure (and clean) `Clock/interval` does not refer to a running interval.

We won't need manual `clear()` if we define our `Interval` models.